lib/plugins/aws/package/compile/events/httpApi/index.js

```diff
@@ -453,6 +454,14 @@ Object.defineProperties(
     compileIntegration: d(function(routeTargetData) {
       const providerConfig = this.serverless.service.provider;
       const userConfig = providerConfig.httpApi || {};
+
+      if (!userConfig.payload) {
+        logDeprecation(
+          'AWS_HTTP_API_VERSION',
+          'Default HTTP API Payload version will be switched to 2.0 with next major release. Configure "payload" explicitly to hide this message.'
```

Comment:
Let's put "httpApi.payload" instead of just "payload" (that'll reduce confusion on where setting should be placed)